src/components/modals/RoleModal.vue

```diff
@@ -72,6 +103,13 @@ export default {
       }
       this.$store.commit("toggleModal", "role");
     },
+    isBluffs() {
+      return this.playerIndex >= 0;
```

Comment:
That's not a `method`, but a `computed` function at best, and since it is only used once, I would inline it into the template. :-)